firehydrant/slack_channels.go

```diff
@@ -40,17 +53,19 @@ func (c *RESTSlackChannelsClient) Get(ctx context.Context, slackID string) (*Sla
 	}
 
 	if channels.Channels == nil || len(channels.Channels) == 0 {
-		return nil, fmt.Errorf("no slack channel found with id '%s'", slackID)
+		return nil, fmt.Errorf("no slack channel found with options: name: %s, id: %s", params.Name, params.ID)
 	}
 	if channelCount := len(channels.Channels); channelCount > 1 {
 		// "at least" because it may paginate.
 		tflog.Error(ctx, "found more than one Slack channel", map[string]interface{}{
-			"id":    slackID,
-			"found": channelCount,
+			"query_id":   params.ID,
+			"query_name": params.Name,
```

Comment:
let's keep this one as "id" and "name" (also below) — "query" has a different meaning and refers to a different parameter in our API.